dotnet/src/Connectors/Connectors.AzureOpenAI/Extensions/AzureOpenAIServiceCollectionExtensions.cs

```diff
@@ -234,6 +235,109 @@ public static class AzureOpenAIServiceCollectionExtensions
 
     #endregion
 
+    #region Images
+
+    /// <summary>
+    /// Add the Azure OpenAI text-to-image service to the list.
```

Comment:
I'm guessing we have this `to the list` text everywhere, but it seems a bit vague.  Should we update it to say to the service collection or to the kernel as appropriate?

Reply:
I've collected a bunch of small improvements and fixes in my TODO list and will rephrase the XML document comments in a separate PR, if you don't mind.

Reply:
Sure, no problem.